src/curriculum/stretching-and-shrinking.json

```diff
@@ -2,18 +2,63 @@
   "title": "Stretching and Shrinking",
   "subtitle": "Understanding Similarity",
   "lookingAhead": {
-    "content": [
-      "A map is a scale drawing of the place it represents. You can use a map to find actual distances to any place in the world. How can you estimate the distance from Cape Town, South Africa to Port Elizabeth, South Africa?",
-      "Suppose that you want to find the width of a river that is too wide to measure directly. How can you use similar triangles to find the distance across the river?",
-      "Here is a picture of Duke, a real dog. If you know the scale factor between Duke and the picture, how can you determine how long Duke is from his nose to the tip of his tail?",
-      "You probably use the word similar in everyday conversation. For example, you might say that one song sounds similar to another song. You may also say that your friend’s bike is similar to yours. In many cases, you might use the word similar to describe objects and images that are the same shape but not the same size.",
-      "A floor plan of a house has the same shape as the actual house, but it is much smaller. The images on a movie screen as the same shape as the real people and objects they depict, but they are much larger. You can order your school portrait in a variety of sizes, but your face will have the same shape in each photo. In the Shapes and Design units you studied a variety of shapes, including the properties that make some shapes important in science, engineering, construction, and art.",
-      "What features of shapes are important in determining the unique shape, like polygons?",
-      "When you encounter a new problem, it is a good idea to ask yourself questions. In this Unit, you might ask questions such as:",
-      "What determines whether two shapes are similar?",
-      "What is the same and what is different about two similar figures?",
-      "When figures are similar, how are the side lengths, areas, and scale factors related?",
-      "How can I use similar figures to find missing measurements?"
+    "tiles": [
+      {
+        "content": {
+          "type": "Text",
```

Comment:
I'm wondering - should we switch to use camelCase for the tool type constant values since we are using camelCase for the key values in the json?  Its more of a style question as this looks a little off.

Reply:
It obviously doesn't make much difference. I've been consciously trying to make this an author-friendly JSON format and I was thinking of those as the titles of the tools, which seemed like they should naturally be capitalized, but I don't feel strongly about it.